source/common/runtime/runtime_features.cc

```diff
@@ -80,6 +80,8 @@ RUNTIME_GUARD(envoy_reloadable_features_validate_connect);
 RUNTIME_GUARD(envoy_reloadable_features_validate_upstream_headers);
 RUNTIME_GUARD(envoy_restart_features_explicit_wildcard_resource);
 RUNTIME_GUARD(envoy_restart_features_no_runtime_singleton);
+RUNTIME_GUARD(envoy_restart_features_remove_runtime_singleton);
```

Comment:
Is this correct? The line above seems to clash with the previous one? Not sure how harm this is though...

Reply:
Probably not correct, but it should be harmless as only the no_runtime_singleton is used in the code.